bundles/org.openhab.binding.twitter/README.md

```diff
@@ -0,0 +1,66 @@
+# Twitter Binding
+
+The Twitter binding allows your home to Tweet, 280 characters at a time. It also supports direct messages and tweeting with media.
+
+## Supported Things
+
+```
+Account - Twitter Account.
+
+```
+
+## Thing Configuration
+
+The Twitter Account Thing requires you create a Twitter App in the Twitter Developer Page.
+
+|   Property        | Default | Required | Description                       |
+|-------------------|---------|:--------:|-----------------------------------|
+| consumerKey       |         |   Yes    | Consumer API Key                  |
+| consumerSecret    |         |   Yes    | Consumer API Secret               |
+| accessToken       |         |   Yes    | Access Token                      |
+| accessTokenSecret |         |   Yes    | Access Token Secret               |
+| refresh           | 30      |   No     | Tweet refresh interval in minutes |
+
+
+## Channels
+
+```
+Last Tweet - Latest Tweet.
```

Comment:
You could mention the Item Type, the Channel Type ID and the description by a table, also to keep the readme consistent with other bindings. See https://www.openhab.org/addons/bindings/telegram/#channels for example.